File: mps-cli-py/model/SLanguage.py

class SLanguage:

    def __init__(self, name, uuid):
        self.name = name
        self.uuid = uuid
        self.concepts = []


    def find_concept_by_name(self, name):
        for c in self.concepts:
            if c.name == name:
                return c
        return None

File: mps-cli-rs/src/model/slanguage.rs
use crate::model::sconcept::SConcept;
use std::cell::RefCell;
use std::rc::Rc;

pub struct SLanguage {
    pub name: String,
    pub id: String,
    pub concepts: Vec<Rc<SConcept>>,
}

impl SLanguage {
    pub fn new(name: String, id: String) -> Self {
        SLanguage {
            name,
            id,
            concepts: vec![],
        }
    }
}
